src/components/Counter/Counter.visual.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Counter } from './';
+import { VisualContainerElement } from '../../../test/visual/VisualContainerElement';
+
+class CounterVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualContainerElement>
+        <Counter
+          inputAriaLabel={'amount'}
+          incrementAriaLabel={'increment'}
+          decrementAriaLabel={'decrement'}
+          onChange={val => console.log(val)}
+          {...this.props}
+        />
+      </VisualContainerElement>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+    ],
```

Comment:
Let's add some more visual tests:
for `disabled`, `error`, and disabled `+` and disabled `-`

Reply:
done